Sources/BraveWallet/Crypto/NFT/NFTView.swift

```diff
@@ -156,44 +167,57 @@ struct NFTView: View {
     return attributedString
   }
   
-  var body: some View {
-    ScrollView {
-      VStack {
-        nftHeaderView
-        if nftStore.userVisibleNFTs.isEmpty {
-          emptyView
-            .listRowBackground(Color(.clear))
-        } else {
-          LazyVGrid(columns: nftGrids) {
-            ForEach(nftStore.userVisibleNFTs) { nft in
-              Button(action: {
-                selectedNFTViewModel = nft
-              }) {
-                VStack(alignment: .leading, spacing: 4) {
-                  nftImage(nft)
-                    .padding(.bottom, 8)
-                  Text(nft.token.nftTokenTitle)
-                    .font(.callout.weight(.medium))
-                    .foregroundColor(Color(.braveLabel))
-                    .multilineTextAlignment(.leading)
-                  if !nft.token.symbol.isEmpty {
-                    Text(nft.token.symbol)
-                      .font(.caption)
-                      .foregroundColor(Color(.secondaryBraveLabel))
-                      .multilineTextAlignment(.leading)
+  @ViewBuilder var nftGridsView: some View {
+    if nftStore.userVisibleNFTs.isEmpty {
+      emptyView
+        .listRowBackground(Color(.clear))
+    } else {
+      LazyVGrid(columns: nftGrids) {
+        ForEach(nftStore.userVisibleNFTs) { nft in
+          Button(action: {
+            selectedNFTViewModel = nft
+          }) {
+            VStack(alignment: .leading, spacing: 4) {
+              nftImage(nft)
+                .contextMenu {
```

Comment:
Should the context menu be on the `VStack` in the Button instead of just the image so the name is also visible?
![Simulator Screenshot - iPhone 14 Pro - 2023-06-05 at 13 15 14](https://github.com/brave/brave-ios/assets/5314553/72112e07-8906-4527-b39d-20b608b4e46c)

Reply:
i thought only highlight the image looks better 😅